static/app/components/errorRobot.tsx

```diff
@@ -42,6 +44,10 @@ class ErrorRobot extends Component<Props, State> {
 
   componentDidMount() {
     this.fetchData();
+    logExperiment({
```

Comment:
@Neo-Zhixing I think we need to gate this until we've updated the sandbox with the header changes. So either don't log the experiment or only do it behind a feature flag.

Reply:
Let's just delay merging this PR until the sandbox PR merges